package.json

```
@@ -50,7 +50,7 @@
   },
   "dependencies": {
     "@patternfly/patternfly": "4.224.2",
-    "@patternfly/quickstarts": "^2.4.1",
+    "@patternfly/quickstarts": "^2.4.3",
```

```suggestion
    "@patternfly/quickstarts": "2.4.3",
```

---

Maybe?

```suggestion
    "@patternfly/quickstarts": "~2.4.3",
```
I think it's ok to accept new patches

---

We've had many problems with this in the past, updates in the shadow that brought fatal collateral damage. Since then I decided we would stick to fixed versions, let Snyk or whatever tool tell us to update proactively.